lib/login_gov/hostdata.rb

```diff
@@ -36,7 +36,8 @@ def self.instance_role
     end
 
     def self.in_datacenter?
-      File.directory?(CONFIG_DIR)
+      return @in_datacenter unless @in_datacenter.nil?
```

Comment:
Great optimization!

IMO a better check for truthy/falsy values is `defined?` because then if the value ever was literal `false` it would mess up this check as written.

So I usually write:

```rb
return @in_datacenter if defined?(@in_datacenter)
```